code/src/main/java/com/onlinecontacttracing/messaging/SystemMessage.java

```diff
@@ -0,0 +1,23 @@
+package com.onlinecontacttracing.messaging;
+
+public enum SystemMessage implements HasEnglishTranslation, HasSpanishTranslation  {
+  VERSION_1("Hello, you are receiving this message because someone you know has tested positive for COVID-19. Please consider quarantining and self-isolating for 14 weeks in addition to getting tested.",
```

Comment:
This one too. How are you expecting this code to be consumed that you're using an enum here?